src/pudl/package_data/meta/xlsx_maps/eia860/column_maps/plant.csv

```diff
@@ -1,46 +1,48 @@
+iso_rto,,,,,,,iso_rto,iso_rto,iso_rto,,,,,,,
+iso_rto_code,,,,,,,iso_rto_code,iso_rto_code,iso_rto_code,,,,,,,
+latitude,,,,,,,,,latitude,latitude,latitude,latitude,latitude,latitude,latitude,latitude
+longitude,,,,,,,,,longitude,longitude,longitude,longitude,longitude,longitude,longitude,longitude
+balancing_authority_code_eia,,,,,,,,,,balancing_authority_code,balancing_authority_code,balancing_authority_code,balancing_authority_code,balancing_authority_code,balancing_authority_code,balancing_authority_code
+balancing_authority_name_eia,,,,,,,,,,balancing_authority_name,balancing_authority_name,balancing_authority_name,balancing_authority_name,balancing_authority_name,balancing_authority_name,balancing_authority_name
+grid_voltage_kv,,,,,,,,gridvoltage,grid_voltage_kv,grid_voltage_kv,grid_voltage_kv,grid_voltage_kv,grid_voltage_kv,grid_voltage_kv,grid_voltage_kv,grid_voltage_kv
+grid_voltage_2_kv,,,,,,,,,,grid_voltage_2_kv,grid_voltage_2_kv,grid_voltage_2_kv,grid_voltage_2_kv,grid_voltage_2_kv,grid_voltage_2_kv,grid_voltage_2_kv
+grid_voltage_3_kv,,,,,,,,,,grid_voltage_3_kv,grid_voltage_3_kv,grid_voltage_3_kv,grid_voltage_3_kv,grid_voltage_3_kv,grid_voltage_3_kv,grid_voltage_3_kv
+ash_impoundment,,,,,,,,,,ash_impoundment,ash_impoundment,ash_impoundment,ash_impoundment,ash_impoundment,ash_impoundment,ash_impoundment
+ash_impoundment_lined,,,,,,,,,,ash_impoundment_lined,ash_impoundment_lined,ash_impoundment_lined,ash_impoundment_lined,ash_impoundment_lined,ash_impoundment_lined,ash_impoundment_lined
+ash_impoundment_status,,,,,,,,,,ash_impoundment_status,ash_impoundment_status,ash_impoundment_status,ash_impoundment_status,ash_impoundment_status,ash_impoundment_status,ash_impoundment_status
+energy_storage,,,,,,,,,,,,,energy_storage,energy_storage,energy_storage,energy_storage
+natural_gas_pipeline_name_1,,,,,,,,,,natural_gas_pipeline_name,natural_gas_pipeline_name,natural_gas_pipeline_name,natural_gas_pipeline_name_1,natural_gas_pipeline_name_1,natural_gas_pipeline_name_1,natural_gas_pipeline_name_1
+natural_gas_pipeline_name_2,,,,,,,,,,,,,natural_gas_pipeline_name_2,natural_gas_pipeline_name_2,natural_gas_pipeline_name_2,natural_gas_pipeline_name_2
+natural_gas_pipeline_name_3,,,,,,,,,,,,,natural_gas_pipeline_name_3,natural_gas_pipeline_name_3,natural_gas_pipeline_name_3,natural_gas_pipeline_name_3
+pipeline_notes,,,,,,,,,,,,,pipeline_notes,pipeline_notes,pipeline_notes,pipeline_notes
+natural_gas_local_distribution_company,,,,,,,,,,,,,natural_gas_ldc_name,natural_gas_ldc_name,natural_gas_ldc_name,natural_gas_ldc_name
+natural_gas_storage,,,,,,,,,,,,,natural_gas_storage,natural_gas_storage,natural_gas_storage,natural_gas_storage
+liquefied_natural_gas_storage,,,,,,,,,,,,,liquefied_natural_gas_storage,liquefied_natural_gas_storage,liquefied_natural_gas_storage,liquefied_natural_gas_storage
+coordinate_system,,,,datum,,,,,,,,,,,,

```

Comment:
Hmm, a datum and a coordinate system are related to each other, but they're not the same thing. Is there a reason why you chose to rename it `coordinate_system` rather than leaving it `datum`?

https://gis.stackexchange.com/questions/664/difference-between-projection-and-datum

Reply:
In looking at the types of values I thought that they were equivalent to coordinate systems and that that word would be more strait forward for users, but I guess not--good catch! I'll change it back.